apps/crn-server/test/helpers/teardown.ts

```diff
@@ -23,9 +23,11 @@ export const teardownHelper = (
   }
 
   return async () => {
-    for (const record of toDelete) {
-      await record.client.delete(record.id);
-    }
+    try {
```

Comment:
hmm the issue with this line is that since I'm deleting the events while the tests are running, in the teardown the events ids are still in the array, so it gives an error because it can't find the record to delete it. Do you think it's a major problem to leave it like that?

Reply:
im not sure i fully understand, technically this shouldn't be a huge deal cause the most likely reason this fails is because the records has already been deleted or is missing, but maybe at least log a warning for some visibility?

Reply:
I think I only understood your comment now after this comment from Lenny https://github.com/yldio/asap-hub/pull/3236#discussion_r1254459362 😅 
The try catch was indeed out of place. I added the log here [review: add log message when deleting fixture fails](https://github.com/yldio/asap-hub/pull/3236/commits/16fa09ffd7de0c77e3f51b0426d167cb294a8ded)